robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -113,9 +98,192 @@ def store_template_record_id(self,template):
+                    try:
+                        self.selenium.click_element(field_loc)
+                    except ElementClickInterceptedException:
+                        self.selenium.execute_javascript("window.scrollBy(0,100)")
+                        self.selenium.click_element(field_loc)    
+                    if placeholder=="Select an Option":
+                        popup=npsp_lex_locators["flexipage-popup"]
+                        self.selenium.wait_until_page_contains_element(popup)
+                        option=npsp_lex_locators["span_button"].format(value)
+                        self.selenium.click_element(option)
+                    elif placeholder=="Search...":
+                        self.salesforce.populate_lookup_field(key,value)
+                    elif "Date" in field:
+                        locator=npsp_lex_locators["bge"]["datepicker_open"].format("Date")  
+                        self.selenium.wait_until_page_contains_element(locator)
+                        self.selenium.click_button(value)    
+                        self.selenium.wait_until_page_does_not_contain_element(locator,error="could not open datepicker")
+                    else:
+                        self.salesforce._populate_field(field_loc,value) 
+    
+    def add_field_bundle_to_new_section(self,bundle):
+        """"""
+        try:
+            self.selenium.click_button("Add Section")
+        except ElementClickInterceptedException:
+            self.selenium.execute_javascript("window.scrollBy(0,100)")
+            self.selenium.click_button("Add Section")
+        checkbox=npsp_lex_locators["gift_entry"]["field_input"].format(bundle,"input")
+        self.selenium.scroll_element_into_view(checkbox)
+        cb_loc=self.selenium.get_webelement(checkbox)
+        if not cb_loc.is_selected():
+            try:
+                self.salesforce._jsclick(checkbox) 
+            except ElementClickInterceptedException:
+                self.selenium.execute_javascript("window.scrollBy(0,100)")
+                self.salesforce._jsclick(checkbox)   
+
+    
+
+@pageobject("Form", "Gift Entry")
+class GiftEntryFormPage(BaseNPSPPage, BasePage):
+
+    def _is_current_page(self):
+        """
+        Verifies that current page is Gift Entry form page
+        """
+        self.selenium.wait_until_page_contains("Donor Information")
+
+    def verify_field_default_value(self,**kwargs):
+        """verifies that the field contains given default value
+        where key=field name and value=default value"""
+        for key,value in kwargs.items():
+            field=npsp_lex_locators["gift_entry"]["field_input"].format(key,"input")
+            self.selenium.wait_until_page_contains_element(field)
+            time.sleep(.5)
+            element=self.selenium.get_webelement(field)
+            print(element)
+            default_value=element.get_attribute("value")
+            self.builtin.log_to_console(f'def value is {default_value}')
```

Comment:
please remove log to console if. it is debugging purpose

Reply:
done